### ISSUE TITLE
MARC conversion on DVD template-- subject problem

### ISSUE THREAD

**Issue by gdegroat** [LD4P/sinopia_editor#2684]
In Sinopia Stage, using Firefox in DVD unnested templates for instance and work:

The MARC conversion worked mostly (i assume special mappings haven't been done yet for this template), however, i think that one of the subject headings should have appeared in the MARC conversion, but didn't.  In the Work template, the third subject, Actresses--Drama, did match with a URI from id.loc.gov, so i think it should have converted to MARC.  My understanding is that the literals are being ignored but this one was not a literal.

![image](https://user-images.githubusercontent.com/44270367/95915727-7372fc80-0d5c-11eb-97b3-9493cb8d8648.png)

**Comment by justinlittman** [comment]
Would you expect http://id.loc.gov/authorities/subjects/sh2009113554 to be a 650? Is that determined by the first of the complex subject (i.e., "Actresses")?

**Comment by gdegroat** [comment]
Yes, the  first element of the complex subject determines the MARC tag.    Hmm, i'm looking at the record for Actresses--Drama in id.loc.gov to see if that can be determined from that record.  If you viewed the MARC record for that heading you would see that the tag was 150, which would be a 650 as a MARC subject.  But i'm not sure how you can determine that from the RDF on the other end of the URI.  

Ok, i'm comparing it to a heading beginning United States--History.  In the eye-readable display i see that United States has a picture of a world next to it, which is probably to indicate that it is a geographic heading (151 in MARC which is 651 in a MARC subject).  In the record for Actresses--Drama,  Actresses has a T next to it, presumably meaning topic (150/650).  The --Drama has a picture of a palette next to it, which i can only think means form/genre since that subfield should be coded $v.  So presumably this info is coded in machine readable form somewhere in there and needs to be detected by the conversion.
